integration-cli/docker_utils_test.go

```diff
@@ -120,6 +120,11 @@ func newRequestClient(method, endpoint string, data io.Reader, ct, daemon string
 
 func deleteContainer(container ...string) error {
```

Comment:
SGTM, but I wonder theoretically there can be a case that really needs to catch the "No such container" error, probably when the `deleteContainer` is called from non-teardown functions.
(I agree we don't have such a case in the current code: https://github.com/docker/docker/search?utf8=%E2%9C%93&q=integration-cli+deleteContainer&type=Code )

So, how about introducing a variable that controls this behavior?
```go
func deleteContainer(ignoreNoSuchContainer bool, container ...string) error
```